web-test-runner.utils.js

```diff
@@ -24,6 +24,21 @@ export const chromium = playwrightLauncher({
         }),
 });
 
+export const coverallsChromium = playwrightLauncher({
+    product: 'chromium',
+    createBrowserContext: ({ browser }) =>
+        browser.newContext({
+            ignoreHTTPSErrors: true,
+            permissions: ['clipboard-read', 'clipboard-write'],
+        }),
+    launchOptions: {
+        // TODO: Remove executablePath when we fix Playwright versions mismatch
```

Comment:
Can we use a jsdoc @todo? https://jsdoc.app/tags-todo

Also is the todo to remove just the executable path or will we eventually be able to remove the custom coverallsChromium?